cmplibrary/src/test/java/com/sourcepoint/cmplibrary/ConsentLibBuilderTest.java

```diff
@@ -0,0 +1,184 @@
+package com.sourcepoint.cmplibrary;
+
+import android.app.Activity;
+import android.view.ViewGroup;
+
+import org.json.JSONObject;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ConsentLibBuilderTest {
+
+
+    private ConsentLibBuilder consentLibBuilder;
+    private ConsentLib.Callback callback;
+
+    @Before
+    public void initConsentLibBuilder() {
+        consentLibBuilder = new ConsentLibBuilder(808, "cybage.com", mock(Activity.class));
+        callback = new ConsentLib.Callback() {
+            @Override
+            public void run(ConsentLib c) {
+
+            }
+        };
+    }
```

Comment:
Really cool, using the `@Before` annotation make the test cases much more readable!
Just 2 small notes:
1. I'd change the account id from `808` to `123` and site name from `cybage.com` to `example.com` just to imply that this code being tested doesn't depend on a specific account id or site name.
2. I think we can refactor the `callback` assignment to pass a lambda instead. Something like:
```java
callback = (ConsentLib c) -> {};
```
Just so we have less code while keeping it still readable :)

Reply:
good suggestion , I would refactor code as suggested.